index.php

```diff
@@ -52,6 +230,55 @@
     2) For a given patient, all suitable matches ranked from worst to best from all the registered hospitals are displayed.</b><br> 
   </h3>
 </div>
+<footer class="footer-distributed">
+
+<div class="footer-left">
+    <img src="images/logo3.png">
+  <h3>About<span>Eduonix</span></h3>
+
+  <p class="footer-links">
+    <a href="#">Home</a>
+    |
+    <a href="#">Blog</a>
+    |
+    <a href="#">About</a>
+    |
+    <a href="#">Contact</a>
```

Comment:
add link here

Reply:
Hey, wouldn't it be better if we put this in a `css` file? So, that we can includes this footer in other pages also